x-pack/legacy/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -188,6 +188,24 @@ export class TimeSeriesExplorer extends React.Component {
    */
   contextChart$ = new Subject();
 
+  /**
+   * Returns field names that don't have a selection yet.
+   */
+  getFieldNamesWithEmptyValues = () => {
+    const latestEntityControls = this.getControlsForDetector();
+    return latestEntityControls
+      .filter(({ fieldValue }) => !fieldValue)
+      .map(({ fieldName }) => fieldName);
+  };
+
+  /**
+   * Checks if all entity control dropdowns have a selection.
+   */
+  arePartitioningFieldsProvided = () => {
```

Comment:
not need in an arrow function, could be a method

Reply:
The whole file has a mix of styles, I'll keep it like that since the "surrounding" functions have a similar style :)